dex-load/src/main/scala/com/wavesplatform/dex/load/TankGenerator.scala

```diff
@@ -25,7 +31,12 @@ object TankGenerator {
   private def mkAssets(count: Int = settings.assets.count): List[String] = {
     println(s"Generating $count assets... ")
     val assets = (1 to count).map(_ => mkAsset()).toList
-    waitForHeightArise()
+    val asset  = assets.get(new Random().nextInt(assets.length - 1))
```

Comment:
If you see the documentation about `nextInt`, you might notice that `nextInt(x)` returns a number in `[0;x)`. So it excludes `x`. The right code here if we need to get the latest asset too:
```
new Random().nextInt(accounts.length)
```

Reply:
ok